app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/MappingWizardClustersStep.js

```diff
@@ -31,36 +23,25 @@ class MappingWizardClustersStep extends React.Component {
       isFetchingSourceClusters,
       sourceClusters,
       isFetchingTargetClusters,
-      targetClusters
+      targetClusters,
+      removeSourceClusters,
+      removeTargetCluster,
+      addTargetCluster,
+      addSourceClusters
     } = this.props;
 
     if (!isFetchingSourceClusters && !isFetchingTargetClusters) {
       return (
-        <div>
-          <p>Source Clusters</p>
-          {sourceClusters.map(cluster => (
-            <div key={cluster.id}>
-              <span>
-                <b>Name</b>: {cluster.name} &nbsp;
-              </span>
-              <span>
-                <b>ID</b>: {cluster.id} &nbsp;
-              </span>
-            </div>
-          ))}
-          <br />
-          <p>Target Clusters</p>
-          {targetClusters.map(cluster => (
-            <div key={cluster.id}>
-              <span>
-                <b>Name</b>: {cluster.name} &nbsp;
-              </span>
-              <span>
-                <b>ID</b>: {cluster.id} &nbsp;
-              </span>
-            </div>
-          ))}
-        </div>
+        <Field
+          name="clusterMappings"
+          component={ClustersStepForm}
+          sourceClusters={sourceClusters}
+          targetClusters={targetClusters}
+          removeSourceClusters={removeSourceClusters}
+          removeTargetCluster={removeTargetCluster}
+          addTargetCluster={addTargetCluster}
+          addSourceClusters={addSourceClusters}
+        />
       );
     }
     return null;
```

Comment:
later... we can totally show a cool `<Spinner>` component...but I am happy with this for now!